src/instructlab/train/linux_train.py

```diff
@@ -194,6 +242,7 @@ def linux_train(
 
     if four_bit_quant:
         print("LINUX_TRAIN.PY: USING 4-bit quantization with BitsAndBytes")
+        use_bf16 = False
```

Comment:
yeah, I was thinking this should go here. I noticed we were doing this already by setting it to !fp16 below (I think?)